interfacetypes/tosca.interfaces.node.lifecycle/Standard/InterfaceType.tosca

```diff
@@ -2,6 +2,8 @@ tosca_definitions_version: tosca_simple_yaml_1_3
 interface_types:
   tosca.interfaces.node.lifecycle.Standard:
     derived_from: tosca.interfaces.Root
+	metadata:
```

Comment:
@nyuuyn please, put the `metadata` on the same level as `derived_from`. See example:
![image](https://user-images.githubusercontent.com/10276774/160616509-b7cdbf9d-6f15-4cf7-bdb4-1afbb66ee9f7.png)

Reply:
fixed